floyd/cli/run.py

```diff
@@ -244,10 +246,6 @@ def run(ctx, gpu, env, message, data, mode, open_notebook, tensorboard, gpup, cp
     floyd_logger.debug("Created job : %s", expt_info['id'])
 
     job_name = expt_info['name']
-    floyd_logger.info("")
-    table_output = [["JOB NAME"], [job_name]]
-    floyd_logger.info(tabulate(table_output, headers="firstrow"))
-    floyd_logger.info("")
```

Comment:
Do we know if there are folks who have automation scripts depending on this output? We might break their workflow by removing this.

Reply:
Good point, it's hard to tell. I changed it to keep the table, but always display the URL like we do in jupyter mode.

Once we have good public API support, we can be open flexible on making change to CLI US.